src/components/Nav.vue

```diff
@@ -1,68 +1,68 @@
 <template>
-  <div class="header-wrapper blue darken-1">
-    <nav class="header container blue darken-1">
-      <div class="logo">
-        <div class="brand-logo">
-          <svg viewBox="0 0 48 48">
-            <path
-              d="M39.13,39V26.18l0,0H32l0,0V43.74l0,0A21.15,21.15,0,0,1,27.57,45a0,0,0,0,1,0,0V32.2l0,0H20.47l0,0V45.29l0,0H16l0,0V35.75l0,0H8.88l0,0v9.54l0,0H2.71l0,0V2.71l0,0h21A21.47,21.47,0,0,1,45.29,24.9,21.24,21.24,0,0,1,39.16,39,0,0,0,0,1,39.13,39Z"
-            />
-          </svg>
-          <span>Dominican Open Source</span>
+  <div class="header-wrapper">
+    <nav>
+      <div class="nav-wrapper blue darken-1">
+        <div class="logo">
+          <div class="brand-logo center">
+            <svg viewBox="0 0 48 48">
+              <path
+                d="M39.13,39V26.18l0,0H32l0,0V43.74l0,0A21.15,21.15,0,0,1,27.57,45a0,0,0,0,1,0,0V32.2l0,0H20.47l0,0V45.29l0,0H16l0,0V35.75l0,0H8.88l0,0v9.54l0,0H2.71l0,0V2.71l0,0h21A21.47,21.47,0,0,1,45.29,24.9,21.24,21.24,0,0,1,39.16,39,0,0,0,0,1,39.13,39Z"
+              />
+            </svg>
+            <span>Dominican Open Source</span>
```

Comment:
We want to hide this when this when the windows width is < ~600px
```suggestion
            <span class="hide-on-small-only">Dominican Open Source</span>
```

Reply:
Done in the last commit.